src/Symfony/Bridge/Twig/UndefinedCallableHandler.php

```diff
@@ -71,6 +72,10 @@ public static function onUndefinedFunction($name)
             return false;
         }
 
+        if (\class_exists(FullStack::class) && 'web-link' === self::$functionComponents[$name]) {
```

Comment:
what about other components needing to be enabled ?

Reply:
For the other functions/filters?

Reply:
yeah, you add a special error message for the full stack case here when forgetting to enable a component (while the previous error message was for the non-fullstack case to tell to install a package). But web-link is not the only component needing to be enabled explicitly in a fullstack project. If we start to handled this case with nicer error messages, we should do it for all components, not only for one.